lib/extras/codec.cc

```diff
@@ -50,10 +50,9 @@ std::string ExtensionFromCodec(Codec codec, const bool is_gray,
     case Codec::kPSD:
       return ".psd";
     case Codec::kUnknown:
+    default:
```

Comment:
I suspect that the purpose of writing the switch in this way was to make it [exhaustive](https://abseil.io/tips/147), i.e. make it a compile-time error not to handle a new case should one be added. For codecs, perhaps it would make sense to keep it that way?

Reply:
Sure, in this case the Enum is not actually a bitstream field Enum anyway.

Reply:
(It seems the default is still there.)

Reply:
Ah right, I missed a spot. Fixed.